### PR TITLE
Internship models

### PR DESCRIPTION
**Pull request by Badatprogrammiing**
models + tabs switching ...

### CODE REVIEW

**Review by jrchoo — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/fields/ApplicationOutcome.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * An outcome for a job application
+ */
+public class ApplicationOutcome {
+
+    private static final List<String> possibleOutcomes =
+            new LinkedList<>(Arrays.asList("follow up", "ghosted", "offered", "rejected", "accepted"));
```
Comment (seewhyjay) [inline src/main/java/seedu/address/model/fields/ApplicationOutcome.java:15]: Why linked list, also can capitalise first letters and `Follow-up`

File: src/main/java/seedu/address/model/fields/Outcome.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.fields;
+
+/**
+ * Represents the possible outcome for an application/task
+ */
+public enum Outcome {
+    FOLLOW_UP, GHOSTED, REJECTED, OFFERED, ACCEPTED;
+
+    @Override
```
Comment (seewhyjay) [inline src/main/java/seedu/address/model/fields/Outcome.java:9]: same, capitalise please

File: src/main/java/seedu/address/model/fields/Pay.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+    private final NumberFormat moneyFormatter;
+
+    /**
+     * Creates a pay object
+     * @param pay the salary amount
+     */
+    public Pay(BigDecimal pay) {
+        requireNonNull(pay);
+        this.pay = pay;
```
Comment (seewhyjay) [inline src/main/java/seedu/address/model/fields/Pay.java:26]: maybe salary

File: src/main/java/seedu/address/model/fields/TaskOutcome.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * An outcome of a task for a job application
+ */
+public class TaskOutcome {
+
+    private static final List<String> possibleOutcomes =
+            new LinkedList<>(Arrays.asList("follow up", "ghosted", "offered", "rejected"));
```
Comment (seewhyjay) [inline src/main/java/seedu/address/model/fields/TaskOutcome.java:15]: same

**Pull request closed without merging**